src/assets/scss/_intro-section.scss

```diff
@@ -51,7 +51,8 @@
     font-size: 18px;
     font-weight: 300;
     letter-spacing: 3px;
-    text-transform: uppercase;
+    text-transform: lowercase;
+    font: courier;
```

Comment:
If you are only putting the font's name, I would suggest using the more precise `font-family` property instead.